### ISSUE TITLE
"Werkstück" 

### ISSUE THREAD

**Issue by meaku** [peerigon/react-and-caviar#2]
Woran wollen wir hacken? Ideen?
- Dashboard

**Comment by mattiloh** [comment]
- **Ein kleines Wiki** im Stil von Github-Wikis. Ganz simpel, mark-down renderer, erst mal ohne Authentifizierung.
- **Gästebuch** (so 90s!) _on steroids_ mit Up- und Down-Votes für die Einträge (à la stackoverflow), Sortierung nach Datum, Name, Voting und Zufall. Da wären evtl. ein paar gute UI-Sachen dabei.
- **Multi-Todo-List**: Mehrere Todo-Lists, zwischen denen man einzelne Todos hin und her schieben kann (könnte tricky werden für eine schnelle Implementierung, wäre aber evtl. von der Flux- und React-Komponenten-Architektur interessant).

Noch andere Ideen?

**Comment by mattiloh** [comment]
Idee von @topa 
- **Web RTC**
- **Twitter Stream**

**Comment by mattiloh** [comment]
Noch eine Idee:
- **Chat**

Gibt's schon Tendenzen? Oder entscheiden wir es am Dienstag?
Ich finde **Web RTC** und **Chat** sehr spannend. Sind aber auch nicht ganz unkompliziert, denke ich. **Twitter Stream** hätte den Vorteil, dass es schon eine fertige API gibt…

**Comment by meaku** [comment]
Vielleicht können wir ja auch zwei simple Beispiele machen. 2x 2er Teams? 

Ich denke WebRTC UND React ist vielleicht etwas viel. Ich hab mit WebRTC mal rumgespielt, aber so richtig fit bin ich damit nicht. Vielleicht wäre ja ein Video Chat auf WebRTC Basis was für den nächsten Hackathon?

**Comment by topa** [comment]
Vielleicht ist die Idee mit den Teams nicht schlecht! Stelle es mir gerade kompliziert vor zu viert an einem kleinen Projekt zu arbeiten.

Paul Torka | +49-8131-779979-3
peerigon UG (haftungsbeschränkt) | Wallachweg 3 | 85221 Dachau | Germany

Am 7. März 2015 bei 12:15:57, Michael (notifications@github.com) schrieb:

Vielleicht können wir ja auch zwei simple Beispiele machen. 2x 2er Teams?

Ich denke WebRTC UND React ist vielleicht etwas viel. Ich hab mit WebRTC mal rumgespielt, aber so richtig fit bin ich damit nicht. Vielleicht wäre ja ein Video Chat auf WebRTC Basis was für den nächsten Hackathon?

**Comment by jhnns** [comment]
Jo, 2 2er Teams sind wahrscheinlich besser. Am Ende kann man ja dann das Ergebnis vorstellen, so dass die anderen auch davon profitieren.

**Comment by mattiloh** [comment]
Find ich auch gut! Ich les mich mal noch tiefer ein, nicht dass mein Partner alles alleine denken muss, da mir die Node- und JS-Basis ja noch ein "bisschen" fehlt im Vergleich zu euch ;)

**Comment by jhnns** [comment]
Boah des geht gar nicht, wenn ich mitm Matti dann zusammen arbeite. Der checkt das nie!!

**Comment by mattiloh** [comment]
I will be prepared :rage4:

Geil, wusste gar nicht, dass emoji auch das Doom-Face hat.
